NAS-103887 clarify jail/plugin config backup. (#1673)
freenas/freenas-docs: userguide/jails.rst

  other and do **not** share any data.

.. note:: iocage jail configs are stored in
   :file:`/mnt/{poolname}/iocage/jails/{jailname}`. When iocage is

>    updated, a backup of the config file is created as

Reviewer: ```suggestion
   updated, the :file:`config.json` configuration file is backed up as
```

   :file:`/mnt/{poolname}/iocage/jails/{jailname}/config_backup.json`.

>    The backup config can be used to restore previous jail settings.

Reviewer: This needs to be more specific. Can the user just rename config_backup.json to config.json to restore the old settings?

Author: yes.
